src/components/presentational/DumbbellChart/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./DumbbellChart";
```

Comment:
You'll also need to add exports to the presentation/index.ts for this to actually be exported from the library